lib/cuda/cudaBasebandBeamformer.cpp

```diff
@@ -0,0 +1,141 @@
+#include "cudaBasebandBeamformer.hpp"
+
+#include "math.h"
+
+using kotekan::bufferContainer;
+using kotekan::Config;
+
+#define CHECK_CU_ERROR(result)                                                                     \
+    if (result != CUDA_SUCCESS) {                                                                  \
+        const char* errstr = NULL;                                                                 \
+        cuGetErrorString(result, &errstr);                                                         \
+        internal_logging(LOG_ERR, __log_prefix, "Error at {:s}:{:d}; Error type: {:s}", __FILE__,  \
+                         __LINE__, errstr);                                                        \
+        std::abort();                                                                              \
+    }
```

Comment:
Why is this not not using `CHECK_CUDA_ERROR` from `cudaUtil.hpp`?

Reply:
I moved this to cudaUtil.hpp -- it's for API functions starting "cu" vs those starting "cuda".  "driver" vs "runtime" API levels.